src/datagrid/plugins/cell-selection.tsx

```diff
@@ -53,23 +52,21 @@ const useOnSelectCell = () => {
         },
         [
             normalizeSelection,
-            normalizeCell,
             afterCellSelection,
             beforeCellSelection,
             onCellSelection,
             setStore,
-            setTableManagerStore,
             tableManagerStore,
         ],
     );
 };
 
 const allowedTargetIds = ['drag-handle'];
 
-const useResetSelectionOnClickOutside = () => {
+const useOnResetSelectionOnClickOutside = () => {
     const { set: setStore } = usePluginsDataStoreRef();
 
-    const onMouseDown = useCallback(
+    return useCallback(
         (e: MouseEvent) => {
             if (allowedTargetIds.includes((e.target as HTMLDivElement)?.id)) return;
```

Comment:
Need to handle mobile

Reply:
We can't handle on mobile, yet.
This function will only run on the web. Won't throw an error on mobile

Reply:
yeah, I meant to check and return, like you were doing in the useEffect that was deleted.

Reply:
I moved it

Reply:
okay

Reply:
It's in [useHandleClickOutside.tsx](https://github.com/webbeetechnologies/bamboo-molecules/pull/249/files#diff-0cf35c807e5df1ef47ccd8f80481c13e57176226452555c849ed67f77833b818)